core/src/banking_stage/committer.rs

```diff
@@ -85,8 +87,9 @@ impl Committer {
                     loaded_accounts_data_size: committed_tx
                         .loaded_account_stats
                         .loaded_accounts_data_size,
+                    result: committed_tx.status.clone(),
                 },
-                Err(_) => CommitTransactionDetails::NotCommitted,
+                Err(err) => CommitTransactionDetails::NotCommitted(err.clone()),
```

Comment:
clones in most cases should be fast - 32 bytes.

only one variant is potentialy slow since it includes a String 🫠

Reply:
That `String` has been removed though, which reduces `TransactionError` size to 12 bytes 🥳 

Will have to wait for sdk version bump to make it to gain the benefit.

https://github.com/anza-xyz/solana-sdk/pull/12